src/pages/questions/ProductInformation.jsx

```diff
@@ -151,17 +153,29 @@ const ProductInformation = ({ question }) => {
                 />
               </Zoom>
               {flagged[src[src.length - 5]] ? (
-                <FlagIcon
-                  onClick={() =>
-                    deleteFlagImage(src, question.barcode, src[src.length - 5])
-                  }
-                />
+                <Tooltip title="Unflag Image">
```

Comment:
Maybe something a bit more informative?, for example I don't have a clear idea of what flagging an image means :(